deployments/mainnet/BaseRegistrarImplementation.json

```diff
@@ -0,0 +1,755 @@
+{
+    "address": "0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147eA85",
+    "abi": [
```

Comment:
Do you need to save the entire abi? You should be able to get the abi from the artefacts when you compiled then just attach an address like this.

```
const EnsRegistry = await (await ethers.getContractFactory("ENSRegistry")).attach(registryAddress)
```